sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CopyAuthorization.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The CopyAuthorization model.
+ */
+@Immutable
+public final class CopyAuthorization {
+
+    /*
+     * Model identifier.
+     */
+    private final String modelId;
+
+    /*
+     * Token claim used to authorize the request.
+     */
+    private final String accessToken;
+
+    /*
+     * Resource Identifier.
+     */
+    private final String resourceId;
+
+    /*
+     * Region of the resource.
+     */
+    private final String resourceRegion;
+
+    /*
+     * The time when the access token expires. The date is represented as the
+     * number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.
+     */
+    private final long expirationDateTimeTicks;
+
+    /**
+     * Create a CopyAuthorization object
+     *
+     * @param modelId The model identifier
+     * @param accessToken The token used to authorize the request
+     * @param resourceId The resource identifier
+     * @param resourceRegion The region of the resource
+     * @param expirationDateTimeTicks The expiry time of the token
+     */
+    public CopyAuthorization(final String modelId, final String accessToken, final String resourceId,
+        final String resourceRegion,
+        final long expirationDateTimeTicks) {
+        this.modelId = modelId;
+        this.accessToken = accessToken;
+        this.resourceId = resourceId;
+        this.resourceRegion = resourceRegion;
+        this.expirationDateTimeTicks = expirationDateTimeTicks;
+    }
+
+    /**
+     * Get the modelId property.
+     *
+     * @return the modelId value.
```

Comment:
in .NET we are only exposing the `ModelId` and the expiration date, as for now, there is no evidence user will like to see/manipulate the rest of the information.
All the other information is only used by the service and it is available to be shared with the convenience method Krista pointed out in a previous comment

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/CopyAuthorization.cs#L14

Reply:
In Java currently, we don't have an efficient way of keeping fields private as they have to be shared across the package to be accessed in the respective client classes.
More discussion on this - https://github.com/Azure/azure-sdk-for-java/issues/10346#issuecomment-620305587

Might investigate this in a separate issue - https://github.com/Azure/azure-sdk-for-java/issues/11476